hq/app/aws/AWS.scala

```diff
@@ -20,4 +33,39 @@ object AWS {
       Failure.awsAccountNotFound(accountId).attempt
     )
   }
+
+  private def clients[T, S <: AwsClientBuilder[S, T]](
```

Comment:
`A` and `B` is a little more conventional in modern Scala but that's a super minor point and I don't mind at all here (just an FYI)

Reply:
Done anyway.  T is conventional in Java.